### PR TITLE
[DOCFIX] Fix `Journal.md` and related strings in code

### PR DESCRIPTION
**Pull request by jenoudet**
See title.

### CODE REVIEW

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -1795,18 +1797,17 @@ public String toString() {
...
           .setScope(Scope.MASTER)
           .build();
   public static final PropertyKey MASTER_EMBEDDED_JOURNAL_ENTRY_SIZE_MAX =
       new Builder(Name.MASTER_EMBEDDED_JOURNAL_ENTRY_SIZE_MAX)
           .setDefaultValue("10MB")
-          .setDescription(String.format(
-              "The maximum single journal entry size allowed to be flushed. "
-              + "This value should be smaller than 30MB. "
-              + "If you update this value, please also update the value of %s ",
-              Name.MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX))
```
Comment (ZhuTopher) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:1809]: Is this comment no longer applicable?

**Review by LuQQiu — APPROVED** [review]
Overall LGTM, thanks for updating it

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (jenoudet) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:1809]: I compared with the description found in the docs and picked whichever one had been updated most recently.

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
@Xenorith cherry-pick?

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**